salt/environment_settings.yml

```diff
@@ -163,7 +166,22 @@ environments:
           cluster_id: xpro-ecommerce
           purpose: xpro-ecommerce
     purposes:
+      xpro-video-qa:
+        app: edx-video-pipeline
+        business_unit: mitxpro
+        domains:
+          - veda-xpro-qa.mitx.mit.edu
+        instances:
+          edx-video-pipeline:
+            number: 1
+            type: t3.medium
+            security_groups:
+              - edx
+          edx-video-worker:
+            number: 2
+            type: t3.large
```

Comment:
Are those numbers and type needed for qa?

Reply:
I'm not sure yet, but given that the workers are actually running the transcode I would assume the answer to be yes.